README.md

```diff
@@ -598,320 +575,6 @@ config :my_app, Oban,
-  use Ecto.Migration
-
-  def up do
-    Oban.Migrations.up(prefix: "private")
-  end
-
-  def down do
-    Oban.Migrations.down(prefix: "private")
-  end
-end
-```
-
-The migration will create the "private" schema and all tables, functions and
-triggers within that schema. With the database migrated you'll then specify the
-prefix in your configuration:
-
-```elixir
-config :my_app, Oban,
-  prefix: "private",
-  repo: MyApp.Repo,
-  queues: [default: 10]
-```
-
-Now all jobs are inserted and executed using the `private.oban_jobs` table. Note
-that `Oban.insert/2,4` will write jobs in the `private.oban_jobs` table, you'll
-need to specify a prefix manually if you insert jobs directly through a repo.
-
-Not only is the `oban_jobs` table isolated within the schema, but all
-notification events are also isolated. That means that insert/update events will
-only dispatch new jobs for their prefix.
-
-### Dynamic Repositories
-
-Oban supports [Ecto dynamic repositories][dynamic] through the
-`:get_dynamic_repo` option. To make this work, you need to run a separate Oban
-instance per each dynamic repo instance. Most often it's worth bundling each
-Oban and repo instance under the same supervisor:
-
-```elixir
-def start_repo_and_oban(instance_id) do
-  children = [
-    {MyDynamicRepo, name: nil, url: repo_url(instance_id)},
-    {Oban, name: instance_id, get_dynamic_repo: fn -> repo_pid(instance_id) end}
-  ]
-
-  Supervisor.start_link(children, strategy: :one_for_one)
-end
-```
-
-The function `repo_pid/1` must return the pid of the repo for the given
-instance. You can use `Registry` to register the repo (for example in the repo's
-`init/2` callback) and discover it.
-
-If your application exclusively uses dynamic repositories and doesn't specify
-all credentials upfront, you must implement an `init/1` callback in your Ecto
-Repo. Doing so provides the Postgres notifier with the correct credentials on
-init, allowing jobs to process as expected.
-
-[dynamic]: https://hexdocs.pm/ecto/replicas-and-dynamic-repositories.html#dynamic-repositories
-
-### Ecto Multi-tenancy
-
-If you followed the Ecto guide on setting up multi-tenancy with foreign keys, you need to add an
-exception for queries originating from Oban. All of Oban's queries have the custom option `oban:
-true` to help you identify them in `prepare_query/3` or other instrumentation:
-
-```elixir
-# Sample code, only relevant if you followed the Ecto guide on multi tenancy with foreign keys.
-defmodule MyApp.Repo do
-  use Ecto.Repo, otp_app: :my_app
-
-  require Ecto.Query
-
-  @impl true
-  def prepare_query(_operation, query, opts) do
-    cond do
-      opts[:skip_org_id] || opts[:schema_migration] || opts[:oban] ->
-        {query, opts}
-
-      org_id = opts[:org_id] ->
-        {Ecto.Query.where(query, org_id: ^org_id), opts}
-
-      true ->
-        raise "expected org_id or skip_org_id to be set"
-    end
-  end
-end
-```
-
-<!-- MDOC -->
```

Comment:
This needs to stay for embedding into the `Oban` module docs.

Reply:
Ooops, this got lost in rebases and crap. Sorry!